modules/items/server.lua

```diff
@@ -72,7 +72,7 @@ CreateThread(function()
 	end
 	Wait(2000)
 	TriggerEvent('ox_inventory:itemList', Items)
-	if Config.DBCleanup then exports.oxmysql:executeSync('DELETE FROM ox_inventory WHERE lastupdated < (NOW() - INTERVAL ? OR data = "[]"', {Config.DBCleanup}) end
+	if Config.DBCleanup then exports.oxmysql:executeSync('DELETE FROM ox_inventory WHERE lastupdated < (NOW() - INTERVAL ? MONTH OR data = "[]")', {Config.DBCleanup}) end
```

Comment:
You cannot put `NOW() - INTERVAL X OR data = "[]"` inside parantheses. This is not valid query.

We are checking **interval or empty data.** 
```sql
WHERE lastupdated < (NOW() - INTERVAL ?) OR data = "[]"
```